src/lib/resource/sections/applicant-have-you-applied-to-us-before.js

```diff
@@ -42,7 +43,17 @@ module.exports = {
             },
             properties: {
                 'q-applicant-have-you-applied-to-us-before': {
-                    title: 'l10nt:q-applicant-have-you-applied-to-us-before.title{?lng,context,ns}',
+                    title: [
+                        '|l10nt',
+                        ['|role.all', 'myself', 'nonDeceased'],
+                        'applicant-have-you-applied-to-us-before.title.myself',
+                        ['|role.all', 'myself', 'deceased'],
+                        'applicant-have-you-applied-to-us-before.title.myself',
```

Comment:
Is this necessary? there are four instances and only three error message variations?
This could be reduced to  ['|role.all', 'myself'],
                        'applicant-have-you-applied-to-us-before.title.myself'

Reply:
Removed